image_width: 1280
image_height: 720
camera_name: narrow_stereo
camera_matrix:
  rows: 3
  cols: 3
  data: [763.836928, 0.000000, 635.352226, 0.000000, 743.109955, 357.577260, 0.000000, 0.000000, 1.000000]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.079337, -0.164708, -0.000922, -0.003387, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.000000, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000, 0.000000, 0.000000, 1.000000]
projection_matrix:
  rows: 3
  cols: 4
  data: [709.822021, 0.000000, 623.774482, 0.000000, 0.000000, 750.064331, 356.566702, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000]
